app/helpers/foreman_ansible/ansible_reports_helper.rb

```diff
@@ -19,6 +19,14 @@ def ansible_module_name(log)
       name
     end
 
+    def ansible_run_in_check_mode?(log)
+      log.message&.value == 't' if check_mode_log?(log)
```

Comment:
`add_reporter_data` sets message value as boolean, but the db column is a text, so I think that is where the `t` comes from. Can we set the value to something more meaningful?

Reply:
@xprazak2 What do you mean by more meaningful? a longer message that says the check mode was enabled?

Reply:
If the message value is a string, I do not think we should assign a boolean. Any reasonable string value would be ok, for example `check_mode_enabled`.

Reply:
Sounds good, I will fix this.